Index: src/main/java/com/example/dao/CustomerRestRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/example/dao/CustomerRestRepository.java	(date 1477662804000)
+++ src/main/java/com/example/dao/CustomerRestRepository.java	(revision )
@@ -10,9 +10,12 @@
  */
 
 
-@RepositoryRestResource(collectionResourceRel = "customer", path = "people")
+@RepositoryRestResource(collectionResourceRel = "customer", path = "customers")
 public interface CustomerRestRepository extends PagingAndSortingRepository<Customer, Long> {
 
-//    List<Customer> findByLastName(@Param("name") String name);
+    List<Customer> findByName(@Param("name") String name);
+    List<Customer> findByNameContaining(@Param("name") String name);
+    List<Customer> findByTelContaining(@Param("tel") String name);
+    List<Customer> findByGender(@Param("gender") String gender);
 
 }
\ No newline at end of file
